addons/sale/static/src/js/upload_rfq_cog_menu/upload_rfq_cog_menu.js

```diff
@@ -0,0 +1,27 @@
+import { registry } from '@web/core/registry';
+import { DocumentFileUploader } from '@account/components/document_file_uploader/document_file_uploader';
+
+const cogMenuRegistry = registry.category('cogMenu');
+
+/**
+ * 'Upload Request for Quotation' Menu
+ *
+ * This menu allows users to import requests for quotation.
+ */
+export class QuotationRequestUploader extends DocumentFileUploader {
+    static template = 'upload_rfq_cog_menu.QuotationRequestUploader';
+
+    getResModel() {
+        return 'sale.order';
+    }
+}
+
+export const quotationUploaderMenuItem = {
+    Component: QuotationRequestUploader,
+    groupNumber: 0,
+    isDisplayed: ({ config, searchModel }) =>
+        searchModel.resModel === 'sale.order'
+        && ['list', 'kanban'].includes(config.viewType),
```

Comment:
Should we also need to check Create=true in this action :thinking: As there are some menus where we only display SOs and now allow to create SOs

Like `Order to invoice` menu where upload button was not visible before but this action is visible
![image](https://github.com/user-attachments/assets/9ee2976b-9b2b-43a1-a4f1-566d99b09036)